docs/custom-flows/passkeys.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Passkeys
+description: Learn how to allow your users authenticate with passkeys with Clerk.
+---
+
+# Passkeys
+
+<Callout type="Info">
+  Passkey user sign-in is available on Pro plans. [Upgrade](/pricing) to enable this feature.
+</Callout>
+
+A passkey is a type of sign-in credential that requires one user action, but uses two authentication factors:
+  1. A pin number or biometric data
+  1. A physical device
+
+Because of this, passkeys are generally more secure than passwords, and you should allow users to sign into your Clerk app with them. You can do so with the prebuilt [`<SignIn>`](/docs/components/authentication/sign-in) component, or with Clerk's JS SDK.
+
+<Callout type="Info">
+  Users can't create passkeys with the `<SignUp>` component.
+</Callout>
```

Comment:
Fix introductory paragraph to look similar to https://docs-preview-702.clerkpreview.com/docs/custom-flows/email-sms-otp#create-the-otp-sign-up-flow

```suggestion
```